services/harmony/app/frontends/service-image-tags.ts

```diff
@@ -490,13 +497,15 @@ export async function updateServiceImageTag(
   }
 
   const { service } = req.params;
-  const { tag } = req.body;
+  const { tag, test } = req.body;
```

Comment:
I used `test` because that is what stated in the ticket. The variable I ended up using in the response is `regressionImageTag`. I think I am good with either `regressionTestVersion` or `testVersion` based on your suggestion. Going to pull the team.